### PR TITLE
Fix fuzzy data for various tests

### PR DESCRIPTION
**Pull request by nt1m**

### CODE REVIEW

**Comment by nt1m** [comment]
The checks show that results are improved with this new data.

File: css/css-images/image-orientation/image-orientation-background-properties-border-radius.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 <link rel="author" title="Stephen Chenney" href="mailto:schenney@chromium.org">
 <link rel="help" href="https://drafts.csswg.org/css-images-3/#propdef-image-orientation">
 <link rel="match" href="reference/image-orientation-background-properties-border-radius-ref.html">
-<meta name=fuzzy content="10;100">
+<meta name=fuzzy content="0-3;0-331">
```
Comment (mrobinson) [inline css/css-images/image-orientation/image-orientation-background-properties-border-radius.html:9]: It look like this range doesn't include the original maximum difference?

File: density-size-correction/density-corrected-image-svg-aspect-ratio.html [hunk 2]
```diff
@@ -4,7 +4,6 @@
     <link rel="author" title="Noam Rosenthal" href="noam@webkit.org">
     <link rel="match" href="density-corrected-image-svg-aspect-ratio-ref.html" />
     <meta name="assert" content="Assert that SVG aspect ratio is applied correctly on density-corrected images">
-    <meta name=fuzzy content="10;100">
```
Comment (mrobinson) [inline density-size-correction/density-corrected-image-svg-aspect-ratio.html:7]: Why is it okay to remove the fuzziness now?

**Comment by nt1m** [comment]
@mrobinson My new fuzzy data was based on the wpt.fyi results. It showed that some of these ranges were unnecessarily wide or simply unnecessary. You can check the improved results in the checks tab: https://github.com/web-platform-tests/wpt/pull/34028/checks

(It's mostly Chrome which benefits, but Safari & Firefox do a bit as well)

File: css/css-images/image-orientation/image-orientation-background-properties-border-radius.html (hunk 1 above)
Comment (nt1m) [reply css/css-images/image-orientation/image-orientation-background-properties-border-radius.html:9]: I took the maximum values from https://wpt.fyi/results/css/css-images/image-orientation/image-orientation-background-properties-border-radius.html?label=master&label=experimental&product=chrome&product=webkitgtk&product=firefox&product=safari&aligned&q=css%2Fcss-images%2Fimage-orientation%2Fimage-orientation-background-properties-border-radius.html

File: density-size-correction/density-corrected-image-svg-aspect-ratio.html (hunk 2 above)
Comment (nt1m) [reply density-size-correction/density-corrected-image-svg-aspect-ratio.html:7]: This data for density-size-correction tests was simply wrong, the difference is much higher, so they'll fail as before, but without exact bogus values.

https://wpt.fyi/results/density-size-correction?label=master&label=experimental&product=chrome&product=webkitgtk&product=firefox&product=safari&aligned&q=density-size-correction

**Review by mrobinson — APPROVED** [review]

**Pull request merged**